src/app/teams/teams-member.component.ts

```diff
@@ -12,6 +12,14 @@ import { UserProfileDialogComponent } from '../users/users-profile/users-profile
     .mat-list-item-disabled {
       background-color: white;
     }
+    .mat-card-tasks {
+      padding-top: 20px;
+    }
+    .mat-caption {
+      font-size: 16px;
+      font-weight: bold;
+    }
+    .mat
```

Comment:
I think you can remove, since you didn't specify anything, but other than that lgtm